crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -510,16 +532,116 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>>
                         ))?
                     }
                 }
-            }
-            Some(api_models::payments::MandateReferenceId::ConnectorMandateId(
-                connector_mandate_id,
-            )) => (
-                None,
-                Some(ProcessingOptions {
-                    is_subsequent_auth: true,
+            }?,
+        };
+        Ok(Self {
+            create_transaction_request: AuthorizedotnetPaymentsRequest {
+                merchant_authentication,
+                ref_id,
+                transaction_request,
+            },
+        })
+    }
+}
+
+impl
+    TryFrom<(
+        &AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>,
+        String,
+    )> for TransactionRequest
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        (item, network_trans_id): (
+            &AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>,
+            String,
+        ),
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            transaction_type: TransactionType::try_from(item.router_data.request.capture_method)?,
+            amount: item.amount,
+            currency_code: item.router_data.request.currency.to_string(),
+            payment: Some(match item.router_data.request.payment_method_data {
+                domain::PaymentMethodData::Card(ref ccard) => {
+                    PaymentDetails::CreditCard(CreditCardDetails {
+                        card_number: (*ccard.card_number).clone(),
+                        expiration_date: ccard.get_expiry_date_as_yyyymm("-"),
+                        card_code: None,
```

Comment:
Why is this card_code set to None ?

Reply:
`card_code` is not required for payments which are done using `network_trans_id`